src/bin/background-migration/index.ts

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env node -r sucrase/register
+
+import childProcess from 'child_process'
+import { Cleaner } from 'cleaners'
+import { access, mkdir, readFile, writeFile } from 'fs/promises'
+import fetch, { Response } from 'node-fetch'
+import { join, sep as pathSeperator } from 'path'
+import { promisify } from 'util'
+
+import { config } from './config'
+import {
+  asCheckpoint,
+  asScanFile,
+  asScanIndex,
+  Checkpoint,
+  ScanFile,
+  ScanIndex
+} from './types'
+
+const exec = promisify(childProcess.exec)
+
+// Config
+const { dataDir, sshHost, syncServer, migrationEndpoint } = config
+
+// ---------------------------------------------------------------------
+// App
+// ---------------------------------------------------------------------
+
+async function main(): Promise<void> {
+  await setup()
+
+  // Get index
+  const scanIndex = await getScanIndex()
+
+  // Get checkpoint
+  let checkpoint: Checkpoint | null = await getCheckpoint(scanIndex.partitions)
+
+  while (checkpoint !== null) {
```

Comment:
Batch the requests to do 10 at a time. This is to make there a bit more concurrency in the background migrator script.